list-ops/list_ops_test.py

```diff
@@ -0,0 +1,98 @@
+import unittest
+import math
+
+from list_ops import map_src, length_src, reduce_src
+
+
+class ListOpsTest(unittest.TestCase):
+
+    # necessary functions to test map_src
+    def square(self, x):
+        return x**2
```

Comment:
Sorry, I could have written `map_src(lambda x: x**2, [-1, 2, ...])` instead. I just wanted to give a small example on how to use lambdas.